src/appConfig.ts

```diff
@@ -50,4 +50,12 @@ export interface AppConfig {
    * The secret key used to auth against the BattleNet APIs.
    */
   battlenetClientSecret: string;
-}
+  /**
+   * The secret key used to auth against the gw2 APIs.
+   */
+  gw2APIKey: string;
```

Comment:
Is this ArenaNet or GW2 specific?

Suggest being explicit:
```suggestion
  guildWars2APIKey: string;
```